src/components/Card.js

```diff
@@ -113,6 +119,59 @@ export const Card = () => {
             <label for="hide-password-checkbox" className="hide-password">
               Hide password field before printing
             </label>
+
+            <br/><br/>
```

Comment:
I generally try to not use line breaks `<br>`. Instead, you could wrap both the hide-password label/input and encryption-mode labels/inputs in a `<div>` which will give you the block display and also tighten up the space (which looks a bit cleaner IMO).

```html
<div>
  <label for="hide-password-checkbox" ... </label>
  <input
    type="checkbox"
    id="hide-password-checkbox"
    ...
  />
</div>

<div>
  <label ...>Encryption: </label>
  <input
    type="radio"
    id="encrypt-none"
    ...
  />
  ....
</div>
```